tools/ci_build/github/azure-pipelines/win-gpu-ci-pipeline.yml

```diff
@@ -59,3 +57,20 @@ stages:
         ORT_EP_NAME: DML
         MachinePool: onnxruntime-Win2019-GPU-dml
 
+- stage: kernelDocumentation
+  dependsOn: []
+  jobs:
+    - template: templates/win-gpu-ci.yml
+      parameters:
+        BuildConfig: 'RelWithDebInfo'
+        EnvSetupScript: setup_env_cuda_11.bat
+        buildArch: x64
+        additionalBuildFlags: --gen_doc validate --skip_tests --enable_pybind --use_dml --use_cuda --cuda_version=11.4 --cuda_home="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.4" --enable_cuda_profiling --cmake_extra_defines CMAKE_CUDA_ARCHITECTURES=52
+        msbuildPlatform: x64
```

Comment:
Can we also add this to reduce build time?

`--cmake_extra_defines onnxruntime_BUILD_UNIT_TESTS=OFF`

Reply:
Sounds good. Added locally. I'm going to wait a bit before pushing the next iteration, which would cancel the in-progress pipelines, just to see any build errors.

Reply:
Cool, the kernelDocumentation task looks good on the last run (https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=746932&view=results). So pushed the additional CMake defines and fixed the stale `--use_dml` on the CUDA build.